app/helpers/green_lanes_helper.rb

```diff
@@ -1,59 +1,52 @@
 module GreenLanesHelper
   def exemption_checkbox_checked?(category_assessment_id, exemption_code)
-    cas_checked = category_assessments_checked(category_assessment_id)
-
-    return if cas_checked.nil?
-
-    cas_checked.include?(exemption_code)
+    category_assessments_checked(category_assessment_id)&.include?(exemption_code)
   end
 
   def exemption_checkbox_none?(category_assessment_id)
-    cas_checked = category_assessments_checked(category_assessment_id)
-
-    return if cas_checked.nil?
-
-    cas_checked.include?('none')
+    exemption_checkbox_checked?(category_assessment_id, 'none')
   end
 
-  def render_exemptions_or_no_card(category, assessments)
+  def render_exemptions_or_no_card(category, assessments, result)
```

Comment:
passing the result in mainly for cat 3;

There are some cc's which have no cat 1's and they still could be standard category